django_fido/backends.py

```diff
@@ -76,6 +77,62 @@ def get_user(self, user_id):
             return None
 
 
+class Fido2PasswordlessAuthenticationBackend(object):
```

Comment:
There is a lot of duplication of `Fido2AuthenticationBackend` so either create a common ancestor or make this an implementation of `Fido2AuthenticationBackend` and override the `authenticate` function.

Reply:
Haha believe or not, I originally refactored the backend file to have a `BaseFidoAuthentication` class, but then I thought I don't want to go ahead refactoring without consulting you guys first.

Reply:
Removed .idea folder again, must be accidentally committed back during rebase. Also I had a typo in my global .gitignore :(

Reply:
I have refactored to use mixin to reduce some code duplication, I had another look using the 'parent class' concept to have `def authenticate(..)` throws `NotImplementedError` but it didn't seem right to me when I tried to solve the function signature problem with `@overload`, but happy to go with that approach if you think it's better.

Reply:
The `Fido2GeneralAuthenticationBackend` passes the `user` anyway, so you have to change the signaturo for your backend to match.